common/lib/xmodule/xmodule/js/spec/.gitignore

```diff
@@ -7,3 +7,4 @@
 !time_spec.js
 !collapsible_spec.js
 !xmodule_spec.js
+!sequence/display_spec.js
```

Comment:
The `!` means don't ignore, there's a rule higher up in this .gitignore that says to ignore `*.js` in this folder because it used to be all coffee.